implementation/src/main/layout/take-test.fxml

```diff
@@ -16,6 +16,13 @@
             <Text fx:id="descTimeLimit" layoutX="444.0" layoutY="55.0" scaleX="1.0" scaleY="1.0" strokeType="OUTSIDE" strokeWidth="0.0" text="You have 30 minutes" />
           </children>
         </AnchorPane>
+         <Pane fx:id="editQuestionPane" minHeight="0.0" minWidth="0.0" prefHeight="217.0" prefWidth="594.0" visible="false">
```

Comment:
this might conflict with the stuff I did for grading, but we can go back and fix that later